### ISSUE TITLE
Show and hiding sidebar cuts off network view port 

### ISSUE THREAD

**Issue by jvwong** [PathwayCommons/app-ui#1031]
## Environment info

- Browser version (client) : Chrome Version 68.0.3440.106 (Official Build) (64-bit); Firefox Quantum 61.0.2 (64-bit)
- Node.js version (server) : n/a

## Current (buggy) behaviour

Hiding the 'info/legend' sidebar cuts off the network where the sidebar once was.

![image](https://user-images.githubusercontent.com/4706307/44407951-5734f580-a52d-11e8-8a9b-5107ae0a54b2.png)

## Desired behaviour

Should be able to view network within the area where sidebar once was.

## Minimum steps to reproduce

Load example network. Close sidebar. Drag network towards right.

**Comment by jvwong** [comment]
Maybe this will be moot if #1012 is implemented.

**Comment by maxkfranz** [comment]
> Maybe this will be moot if #1012 is implemented.

Yes, but we should have a shortterm fix for the upcoming release.

**Comment by CaitlinOCallaghan** [comment]
Should we have the side bar open when the page is loaded? 

This will fix the problem, and also allow users to access the FAQ before trying to input a gene list

![image](https://user-images.githubusercontent.com/38890251/44413817-4ccd2880-a53a-11e8-893b-a78e544a7ebe.png)

**Comment by CaitlinOCallaghan** [comment]
If we do leave the side bar open on initial loading, then I think we should also render the toolbar in case the user closes the side menu. The only thing is that it gets confusing with 2 input/search/typable bars in such close proximity. 

![image](https://user-images.githubusercontent.com/38890251/44413976-a897b180-a53a-11e8-9b5f-5b3710b3762a.png)

**Comment by maxkfranz** [comment]
I wouldn't worry about the confusion part re. the text boxes at this point in time.  As long as the graph spacing issue is addressed, I think it should be OK.

**Comment by jvwong** [comment]
I don’t understand how having the sidebar open or closed affects this bug

**Comment by CaitlinOCallaghan** [comment]
For the network there is a class `'network-sidebar-open'` that controls the width of the view port. This is currently hard coded to true so that the class is always applied even if the sidebar is closed, which is the bug. 

If we set `'network-sidebar-open': activeMenu !== 'closeMenu'`,  the view port width should change depending on the status of the sidebar (as it does with interactions). But when the activeMenu is closed at the initial rendering, `activeMenu: closeMenu` and then updated to `activeMenu: enrichmentMenu` once the network has been loaded, this class is not properly applied for some odd reason (resulting part of the network to be covered by the sidebar). 

One way around this is to open the side menu from the get-go when the /enrichment page is first loaded.

**Comment by jvwong** [comment]
>If we set 'network-sidebar-open': activeMenu !== 'closeMenu', the view port width should change depending on the status of the sidebar

Tried this and looks good...

> ...But when the activeMenu is .. updated to activeMenu: enrichmentMenu once the network has been loaded, this class is not properly applied for some odd reason...

You might be updating state  *after* the layout is run?


```js
...
const updateNetworkJSON = async () => {
      ...
      cy.layout(_.assign({}, ENRICHMENT_MAP_LAYOUT, { stop: () => {
        this.setState({
          loading: false,
          activeMenu: 'enrichmentMenu',
          invalidTokens: unrecognized,
          openToolBar: true
        });
      }})).run();
    };
...
```
